camel/functions/search_functions.py

```diff
@@ -270,10 +322,31 @@ def summarize_text(text: str, query: str) -> str:
     return response
 
 
+def continue_search(query: str, answer: str) -> bool:
+    r"""Ask LLM whether to continue search or not.
+
+    Args:
+        query (str): Question you want to be answered.
+        answer (str): Answer to the query.
+
+    Returns:
+        bool: True if the user want to continue search, False otherwise.
+    """
+    prompt = TextPrompt("Do you think the ANSWER can answer the QUERY? "
+                        "Use only 'yes' or 'no' to answer\n"
+                        "===== QUERY ====={query}\n\n"
+                        "===== ANSWER ====={answer}")
+    prompt = prompt.format(query=query, answer=answer)
+    reply = prompt_single_step_agent(prompt)
+    if "yes" in str(reply).lower():
+        return True
+    return False
```

Comment:
logic here seems wrong, if we get yes, then we should set continue_search as False